components/flags_ui/resources/brave_flags_overrides.js

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function braveFlagsUIOverrides() {
+  $('channel-promo-beta').innerHTML = `
+      Interested in cool new Brave features? Try our
+      <a href="https://brave.com/download-beta/">beta channel</a>.
+  `
+  $('channel-promo-dev').innerHTML = `
+      Interested in cool new Brave features? Try our
```

Comment:
This sentence is fine with just replacing "Chrome" with "Brave". This might already be translated so don't think we need to modify this any further.

Reply:
FYI, this sentence is not translated. (https://source.chromium.org/chromium/chromium/src/+/master:components/flags_ui/resources/flags.html;l=59)

```
      <!-- Those strings are not localized because they only appear in
           chrome://flags, which is not localized. -->
```